arrow-libs/core/arrow-core-data/src/main/kotlin/arrow/core/Option.kt

```diff
@@ -1024,9 +1028,11 @@ fun <T> Iterable<T>.lastOrNone(predicate: (T) -> Boolean): Option<T> = this.last
 
 fun <T> Iterable<T>.elementAtOrNone(index: Int): Option<T> = this.elementAtOrNull(index).toOption()
 
+@Deprecated(SelectiveDeprecation)
 fun <A, B> Option<Either<A, B>>.select(f: OptionOf<(A) -> B>): Option<B> =
   branch(f.fix(), Some(::identity))
 
+@Deprecated(SelectiveDeprecation)
```

Comment:
`branch` doesn't seem to exist for `0.11.0`

https://github.com/arrow-kt/arrow-core/blob/0.11.0/arrow-core-data/src/main/kotlin/arrow/core/Option.kt